compiler/rustc_feature/src/unstable.rs

```diff
@@ -550,6 +550,8 @@ declare_features! (
     (unstable, object_safe_for_dispatch, "1.40.0", Some(43561)),
     /// Allows using enums in offset_of!
     (unstable, offset_of_enum, "1.75.0", Some(106655)),
+    /// Allows using multiple nested field accesses in offset_of!
+    (unstable, offset_of_nested, "CURRENT_RUSTC_VERSION", Some(106655)),
```

Comment:
Can you please create a new tracking issue instead? Old tracking issues are hard to navigate, so I'd prefer having new gates in new issues.

Reply:
It is already used by `offset_of_enum`... should two new tracking issues be created? Personally I don't care in any direction.

Reply:
I think there is likely to be significant cross-over in discussions between the nested and enum feature gates so one new tracking issue would probably be best - that allows the current one to be closed.

Reply:
I think I'd prefer to have separate (new) issues for `offset_of_nested` and `offset_of_enum`. If only so that they can be more easily stabilized separately.